cmd/bcpd/app/codec.proto

```diff
@@ -23,10 +23,8 @@ import "github.com/iov-one/weave/x/validators/codec.proto";
 message Tx {
   cash.FeeInfo fees = 1;
   repeated sigs.StdSignature signatures = 2;
-  // Preimage for hashlock.
-  bytes preimage = 3;
   // ID of a multisig contract.
-  repeated bytes multisig = 4;
```

Comment:
No need to break this. The idea was to keep all numbers constant.

I would just comment out 3 and write a comment `// 3 removed: previously hashock`. Please keep multisig at 4

Reply:
I've added a general comment about gaps in field sequences as I don't think having a comment per deprecated field is a good maintenance strategy, no reason to pollute the code with deprecated stuff I suppose. Let me know if it's good enough.